src/pages/client-auth.tsx

```diff
@@ -0,0 +1,26 @@
+import { signIn, signOut, useSession } from "next-auth/react"
```

Comment:
Please make a components folder inside the `src` directory and put `client-auth.tsx` there. Otherwise going to `/client-auth.tsx` will render a page since it's within the `pages` directory haha. This was an oversight on my part though.